src/components/Header.jsx

```diff
@@ -0,0 +1,47 @@
+import { Link, NavLink } from "react-router-dom"
+import { useSelector } from 'react-redux'
+
+import '../styles/header.scss'
+
+const Header = ({ searchMovies }) => {
```

Comment:
- Following the Single Responsibility Principle (SRP), we should separate the search bar into its own component.
- First, we should create a `SearchBar.jsx` component.
- Then, we should create a `Navigation.jsx` component.
- Finally, we should update the `Header` component to act as a container that organizes these components.